src/build/__main__.py

```diff
@@ -60,12 +61,14 @@ def _showwarning(
 def _setup_cli() -> None:
     warnings.showwarning = _showwarning
 
-    try:
-        import colorama
-    except ModuleNotFoundError:
-        pass
-    else:
-        colorama.init()  # fix colors on windows
+    if platform.system() == 'Windows':
+        # try to enable colors
+        try:
+            import colorama
+
+            colorama.init()
+        except ModuleNotFoundError:
+            pass
```

Comment:
This is technically a weakness in our tests - we never verify that colorama can actually be missing on Windows.

Reply:
It can not. because it is a required dependency.

https://github.com/pypa/build/blob/4edd5effe9912eee825ba9532255166dadc08a25/setup.cfg#L28-L33

Reply:
Removed exception check,

Reply:
It’s this way for bootstrapping. Please restore it. We can check for it by uninstalling it after installing build, or by checking in the “non installed” check. One of us can look into adding that, doesn’t mean you had to.